Mirror/Runtime/NetworkClient.cs

```diff
@@ -18,6 +18,14 @@ public class NetworkClient
 
         public static bool pauseMessageHandling;
 
+        // how often are we sending ping messages
+        // used to calculate network time and RTT
+        public float pingFrequency = 2.0f;
+        // average out the last few results from Ping
+        public int pingWindowSize = 10;
+
+        private double lastPing = 0;
```

Comment:
remove unnecessary 'private' please :)

Reply:
Also, maybe 'lastPingTime' is better? otherwise it's not obvious that this is a time and not the last ping's result or similar